.github/workflows/pypi-publish.yml

```
@@ -0,0 +1,24 @@
+          python-version: '3.11'
+      - name: Build package
+        working-directory: sdks/python
+        run: |
+          python -m pip install --upgrade pip build
+          python -m build
+      - name: Publish to PyPI
+        uses: pypa/gh-action-pypi-publish@v1.10.1
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
+          packages_dir: sdks/python/dist
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)